cmd/serve/httplib/httplib.go

```diff
@@ -52,6 +52,27 @@ inserts leading and trailing "/" on --baseurl, so --baseurl "rclone",
 --baseurl "/rclone" and --baseurl "/rclone/" are all treated
 identically.
 
+--template allows a user to specify a custom markup template for http
+and webdav serve functions.  The server exports the following markup
+to be used within the template to server pages:
+{{.Name}} The full path of a file/directory.
```

Comment:
Remember the docs get rendered into markdown so I think this section needs to be in

    ```

Because this is already in a \` string for go you'll need to end that add the ` and start it again, soemthing like

    ` + "```" + `